src/member/fixtures.ts

```diff
@@ -0,0 +1,48 @@
+import { MemberStorageItemsResponse } from '@graasp/sdk';
+
+import { PaginationParams } from '../types.js';
+
+export const itemsResponse: MemberStorageItemsResponse = {
```

Comment:
This should not be called `itemsResponse`. Look at how other fixtures are called. It should have something related to member storage files